### PR TITLE
CSTAR-535: Remove contents of log_dir rather than entire directory

### PR DESCRIPTION
**Pull request by csplinter**

### CODE REVIEW

**Comment by nastra** [comment]
+1, lgtm

@mshuler can you also take a look?

File: tool/cstar_perf/tool/fab_common.py [hunk 1]
```diff
@@ -453,15 +453,16 @@ def destroy(leave_data=False, kill_delay=0):
...
     if not leave_data:
-        for d in config['data_file_directories']:
-            fab.run('rm -rf {data}/*'.format(data=d))
-        fab.run('rm -rf {saved_caches_directory}/*'.format(saved_caches_directory=config['saved_caches_directory']))
-        fab.run('rm -rf {commitlog}/*'.format(commitlog=config['commitlog_directory']))
-        fab.run('rm -rf {flushdir}/*'.format(flushdir=config['flush_directory']))
-        if config.get('hints_directory'):
-            fab.run('rm -rf {hints_directory}/*'.format(hints_directory=config.get('hints_directory')))
-        _clean_up_cdc_directories()
-    fab.run('rm -rf {log_dir}'.format(log_dir=config['log_dir']))
```
Comment (csplinter) [inline tool/cstar_perf/tool/fab_common.py]: @mshuler do you know if this removal of the log_dir is purposefully outside of the 'if not leave_data' block?

File: tool/cstar_perf/tool/fab_common.py [hunk 2]
```diff
@@ -461,7 +461,7 @@ def destroy(leave_data=False, kill_delay=0):
         if config.get('hints_directory'):
             fab.run('rm -rf {hints_directory}/*'.format(hints_directory=config.get('hints_directory')))
         _clean_up_cdc_directories()
-    fab.run('rm -rf {log_dir}'.format(log_dir=config['log_dir']))
+        fab.run('rm -rf {log_dir}/*'.format(log_dir=config['log_dir']))
```
Comment (nastra) [inline tool/cstar_perf/tool/fab_common.py]: `leave_data` actually indicates only leaving the data dirs, not the log files. The log files will anyway be preserved at some other point. I would probably correct the indentation here to how it was before.

Option 2 would be to rename `leave_data` to `leave_data_and_logs`

**Comment by mshuler** [comment]
+1

**Pull request merged**